### DISCUSSION TITLE
Avalonia

### DISCUSSION THREAD

**Discussion by akasolace** [ho-dev/HattrickOrganizer#1925] [General]
@wsbrenk @tychobrailleur  Hello, someone reached out to me today regarding HO. He is a student and a with a friend of him they cloned HO in C#/Avalonia.  Plugins were not there but the main screen was here for all I can tell - youth academy was also there. Based on the demo it looked very modern and reactive. Apparently, they started it during last xmas holiday to learn Avalonia framework.  They said their were ready to share the sources but to continue their effort they wanted authorization to monetize an iOS / Android app. I told them I believed this was not possible with the existing license but that they should reach out to you (tbh I am not sure that license restriction was correct considering they ported all the code) So they should contact you and you will see by yourself what they did in 6 months while learning the language!
Best

**Comment by tychobrailleur** [comment]
My understanding of [LGPLv3](https://fossa.com/blog/open-source-software-licenses-101-lgpl-license/) is that the derivative work also needs to be licensed under LGPLv3 (or GPLv3) or later, and retain copyright notice, but there is no restriction for commercial use.

**Comment by wsbrenk** [comment]
If these developers take the license conditions into account, I don't think there's anything wrong with using the HO sources. Making money, as @tychobrailleur says, shouldn't be a problem. I would even like it if we published it together with them in this GitHub organization. We could support each other.

But this event makes another problem clear to me. If they are working with our CHPP licenses, we have a problem that we need to get to grips with.
